astropy/wcs/utils.py

```diff
@@ -458,8 +468,9 @@ def is_proj_plane_distorted(wcs, maxerr=1.0e-5):
 
     """
     cwcs = wcs.celestial
-    return (not _is_cd_orthogonal(cwcs.pixel_scale_matrix, maxerr) or
-            _has_distortion(cwcs))
+    return not _is_cd_orthogonal(cwcs.pixel_scale_matrix, maxerr) or _has_distortion(
+        cwcs
+    )
```

Comment:
What is the line length allowed in astropy? Writing this on a sinle line would be 90 characters long.
```suggestion
    return not _is_cd_orthogonal(cwcs.pixel_scale_matrix, maxerr) or _has_distortion(cwcs)
```

Reply:
Line length decided was 88, I'll add a `# fmt: skip`